src/components/papers/RollingPaperCard.jsx

```diff
@@ -0,0 +1,58 @@
+import EmojiBadge from '@/components/common/badge/EmojiBadge';
+import ProfileGroup from '@/components/common/profile-image/ProfileGroup';
+
+/**
+ * 롤링페이퍼 카드 컴포넌트
+ */
+const RollingPaperCard = ({
+  name = 'Sowon',
+  messageCount = 30,
+  recentMessages = [],
+  reactions = { like: 20, love: 12, sad: 7 },
+  bgColor = 'bg-purple-200',
+  accentColor = 'bg-purple-300/40',
```

Comment:
서버에서 데이터를 받아오는거라 기본 값은 지정안해도 될 것 같아요!
0명 작성했는데 지금 30이 기본값으로 지정되어있어서
30명이 작성했다고 카드에 보여질 것 같습니다! 리액션 부분도 없는데 임의값이 들어가게 될 것 같습니다~~

그리고 accentColor는 서버에서 데이터로 넘어오지 않아서
backgroundColor 값에 따라 accentColor를 결정하는 방식으로 처리해야할 것 같습니다!
그리고 backgroundImageURL이 있을땐 이걸 카드 배경으로 지정하도록 처리해야할 것 같습니다


```suggestion
  id,
  name,
  messageCount,
  recentMessages,
  topReactions, 
  backgroundColor,
  accentColor,
  backgroundImageURL,
```